src/instructions/proctored_exam/ProctoredExamInstructions.test.jsx

```diff
@@ -0,0 +1,296 @@
+    expect(getByTestId('sequence-content')).toHaveTextContent('Sequence');
+  });
+
+  it('Instructions are shown when attempt status is created', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        proctoringSettings: {
+          platform_name: 'Your Platform',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'created',
+        },
+        exam: {
+          is_proctored: true,
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'created',
+          },
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div>Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(getByTestId('exam.VerificationProctoredExamInstructions-continue-button'))
+      .toHaveTextContent('Continue to Verification');
+  });
+
+  it('Instructions are shown when attempt status is ready_to_start', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        proctoringSettings: {
+          platform_name: 'Your Platform',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'ready_to_start',
+        },
+        exam: {
+          is_proctored: true,
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'ready_to_start',
+          },
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div>Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(getByTestId('proctored-exam-instructions-rulesLink'))
+      .toHaveTextContent('Rules for Online Proctored Exams');
+  });
+
+  it('Instructions are shown when attempt status is submitted', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        proctoringSettings: {
+          platform_name: 'Your Platform',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'submitted',
+        },
+        exam: {
+          is_proctored: true,
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'submitted',
+          },
+        },
+      },
+    });
+
+    const { getByTestId } = render(
+      <ExamStateProvider>
+        <Instructions>
+          <div>Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(getByTestId('proctored-exam-instructions-title')).toHaveTextContent('You have submitted this proctored exam for review');
+  });
+
+  it('Instructions are shown when attempt status is ready_to_submit', () => {
```

Comment:
Are we testing the behavior of the continue working / submit buttons?

I'm also looking at the existing view and notice the "continue working" button is conditional based on the remaining time. This might be worth just filing a bug ticket for later

Reply:
I believe we don't have test for those buttons right now, I will create a bug ticket for missed condition and we will add tests for buttons in that PR also